test/acceptance/e2e.html.tpl

```diff
@@ -4,7 +4,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <script>
       // This script hides the map attribution when served locally improving screenshot testing precision.
```

Comment:
This script is not needed anymore.

This was used to conditionally hide the map attribution depending on if the map was displayed on the developer center as an example or we were on a testing env.

Since testing env and examples are separated we could update this to a `<style>` tag where the attribution is always hidden.

Reply:
:+1: